### PR TITLE
docs(deploy): fix typo (#240) [skip ci]

### PR DESCRIPTION
**Pull request by csanda87**
packaje.json to package.json

### CODE REVIEW

**Review by nurlansu — APPROVED** [review]

File: docs/guide/deploy.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ $ heroku open
 
 To deploy your VitePress app with a [Vercel for Git](https://vercel.com/docs/git), make sure it has been pushed to a Git repository.
 
-Go to https://vercel.com/import/git and import the project into Vercel using your Git of choice (GitHub, GitLab or BitBucket). Follow the wizard to select the project root with the project's `packaje.json` and override the build step using `yarn docs:build` or `npm run docs:build` and the output dir to be `./docs/.vitepress/dist`
+Go to https://vercel.com/import/git and import the project into Vercel using your Git of choice (GitHub, GitLab or BitBucket). Follow the wizard to select the project root with the project's `package.json` and override the build step using `yarn docs:build` or `npm run docs:build` and the output dir to be `./docs/.vitepress/dist`
```
Comment (kemila) [inline docs/guide/deploy.md:256]: !

Comment (kemila) [reply docs/guide/deploy.md:256]: 1

Comment (kemila) [inline docs/guide/deploy.md:256]: 1111

**Review by posva — APPROVED** [review]
Thanks!

**Pull request merged**